ansible/playbooks/generate-csrs.yml

```diff
@@ -29,6 +29,11 @@
       owner: redpanda
       group: redpanda
 
+  - name: Install openssl
+    ansible.builtin.package:
+      name: openssl
+      state: present
```

Comment:
Should this be `latest`? Or at least make it configurable so openssl attempts to keep up to date with security patches?